server/pages/index.jade

```diff
@@ -142,5 +142,20 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				script(src=urls[ chunk + '-min' ])
 		script(type='text/javascript')!='window.AppBoot();'
 
+		script.
+			if (window.console) {
+
+				if(true) {
+                    //Browser supports color output
+                    console.log("%cWait! This area is meant for developers!", "color: red;font-size: xx-large");
+				}else{
+					//No color supported
+                    console.log("Wait! This area is meant for developers!");
+				}
+                console.log("If someone told you to type or paste something into this area it could compromise the security of your account.");
+
+            }
```

Comment:
![screen shot 2017-06-26 at 12 48 11 pm](https://user-images.githubusercontent.com/3857633/27550477-dae4f486-5a6d-11e7-918e-9ac0ccc9baf7.png)
The css styled output.
I am on Version 10.0.1 (11602.2.14.0.7) if that makes any difference.